contrib/babelfishpg_common/src/babelfishpg_common.c

```diff
@@ -42,6 +70,25 @@ _PG_init(void)
 	coll_cb_ptr = (collation_callbacks **) find_rendezvous_variable("collation_callbacks");
 	*coll_cb_ptr = get_collation_callbacks();
 
+	DefineCustomStringVariable("babelfishpg_tsql.server_collation_name",
```

Comment:
Yes, This would be major change and IMHO, should be taken up separately. I also need to evaluate complication around such changes. SO better to take it separate.